code/modules/power/apc.dm

```diff
@@ -1195,8 +1195,14 @@
 			charging = 0
 			chargecount = 0
 
-		if(excess >= 5000000 && !shock_proof) //If there's more than 5,000,000 watts in the grid, start arcing and shocking people.
-			if(prob(5))
+		if(excess >= 2500000 && !shock_proof)
+			var/shock_chance = 5 // 5%
+			if(excess >= 5000000)
```

Comment:
instead of nested ifs, couldn't you use a switch for this?

Reply:
I changed to using else if. I don't want to use switch as it would be more lines for the same effect.